com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/Templates/ShaderPass.template

```diff
@@ -275,9 +275,18 @@ Pass
         ApplyDepthOffsetPositionInput(V, surfaceDescription.DepthOffset, GetViewForwardDir(), GetWorldToHClipMatrix(), posInput);
         #endif
 
-        #ifndef SHADER_UNLIT
+        #ifndef SHADER_UNLIT        
+
+        // TODO CORRECT TERRAIN NORMALS
```

Comment:
@SteveTheodore identified that the Y direction of the normals were flipped:
![image](https://user-images.githubusercontent.com/3450690/137820468-f78fc828-d93a-49e8-81ab-26834b32b8a6.png)

Check if the commented code here resolves this. 
Requires an overload of GetNormalWS in MaterialUtilities.hlsl that takes in isFrontFace and tangentToWorld, since FragInputs doesn't have the modifications that Terrain has made to tangentToWorld.